repositories/user_character.go

```diff
@@ -42,3 +44,30 @@ func (r *UserCharacterRepository) GetList(userId string) ([]models.UserCharacter
 
 	return userCharacters, nil
 }
+
+// ガチャ結果をusers_charactersテーブルにバルクインサートする
+func (r *UserCharacterRepository) InsertBulk(userId string, characters []models.UserCharacterInsert) error {
+	if len(characters) == 0 {
+		return nil
+	}
+
+	// クエリのプレースホルダーを生成
+	valueStrings := make([]string, 0, len(characters))
+	valueArgs := make([]interface{}, 0, len(characters)*3)
```

Comment:
[nits]
Go1.23あたりから `interface` → `any` という型エイリアスが推奨になっています！

```suggestion
	valueArgs := make([]any{}, 0, len(characters)*3)
```

Reply:
https://github.com/ShinnosukeSuzuki/techtrain-mission-ca-tech-dojo-golang/commit/037057034130c56ff7e051acca0eddf0ba7515d9
上記コミットで対応しました。